modules/upgrade/pages/proxy-x.adoc

```diff
@@ -26,180 +19,100 @@ The distribution must be based on {sle}{nbsp}15 SP1, because {productname}{nbsp}
 
 
 .Procedure: Preparing Installation Media
-. On the {productname} Server, create a local directory for the SLE{nbsp}15 installation media.
+. On the {productname} Server, create a local directory for the SLE{nbsp}15 installation media:
++
+----
+mkdir -p /srv/images/sle15sp1
+----
 . Download an ISO image with the installation sources, and mount the ISO image on your server:
 +
 ----
-mkdir /mnt/sle15sp1
-mount -o loop DVD1.iso /mnt/sle15sp1
+mount -o loop DVD1.iso /mnt/
+----
+. Copy everything from the mounted ISO to your local filesystem:
++
+----
+cp -r /mnt/* /srv/images/sle15sp1
```

Comment:
forgot that after this the ISO is not needed anymore. So we can `umount /mnt` .
Not sure if this is needed to say.